src/Coders/Model/Relations/BelongsToMany.php

```diff
@@ -48,36 +50,36 @@ class BelongsToMany implements Relation
      * @param \Illuminate\Support\Fluent $referenceCommand
      * @param \Reliese\Coders\Model\Model $parent
      * @param \Reliese\Coders\Model\Model $pivot
-     * @param \Reliese\Coders\Model\Model $reference
+     * @param \Reliese\Coders\Model\Model $related
      */
     public function __construct(
         Fluent $parentCommand,
         Fluent $referenceCommand,
         Model $parent,
         Model $pivot,
-        Model $reference
+        Model $related
```

Comment:
Why did you renamed this property? 🤔

Reply:
For universality, because in all others relations - this property named 'related'.
I inject trait with comment() getter, who use this property.
Otherwise this trait isn't working and we need add getter in each relation.